getting-started/cloudflare-workers.md

```diff
@@ -126,6 +126,44 @@ bun run deploy
 
 That's all!
 
+### Deploy from Github Action
```

Comment:
Can you move this section to the bottom? I think the other sections are more important than this.

Reply:
@yusukebe its done!